include/test/miral/test_display_server.h

```diff
@@ -93,6 +93,9 @@ struct TestDisplayServer : private TestRuntimeEnvironment
     virtual auto
     build_window_manager_policy(WindowManagerTools const& tools) -> std::unique_ptr<WindowManagementPolicy>;
 
+    /// Wrapper to gain access to the MirRunner
```

Comment:
I note that `invoke_tools` says that this must be called after `start_server()`; presumably this does as well?

Reply:
Fixing